src/components/atoms/Table.module.css

```diff
@@ -23,6 +23,10 @@
   color: var(--color-secondary);
 }
 
+div [class*='rdt_TableCell'] {
+  min-width: 80px;
```

Comment:
for me, only using `min-width: 0` here solves most scrollbar problems in Safari and Firefox. I think this was the intention of the basic reset done further up this file in line 5 & 6 where we have multiple selectors targeting the `role`. I would suggest adding the table cells selector there too, but also selected by `role` so in this case `.table [role='gridcell']`